BackEnd/src/main/java/com/unosquare/admin_core/back_end/controller/DashboardController.java

```diff
@@ -98,6 +96,7 @@ private void mapEmployeeSnapshotViewModel(EmployeeSnapshotDto m, EmployeeSnapsho
         employee.setName(m.getName());
         employee.setEmployeeId(m.getEmployeeId());
```

Comment:
m is a horrible variable name, also this should definitely be within modelmapper.